src/ca_biositing/pipeline/tests/USDA/test_usda_comprehensive.py

```
@@ -0,0 +1,533 @@
+            "domain_desc": "TOTAL",
+            "format": "JSON"
+        }
+
+        try:
+            response = requests.get(BASE_URL, params=params, timeout=TIMEOUT)
+            response.raise_for_status()
+
+            # Handle empty responses that can't be parsed as JSON
+            if not response.text.strip():
+                if commodity == "TOMATOES" and county == "077":
+                    print(f"\n🔍 DEBUG - TOMATOES County 077 (Empty Response):")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)